modules/ROOT/pages/api-products-creating-products.adoc

```diff
@@ -0,0 +1,35 @@
+= Creating API Products
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You create a Product to bundle a set of APIs to enable ease-of-use and efficiency for consumers. To create a bundle, you first create a Product and then add the relevant API instances to it. If you require your Product to be available for public consumption, you must publish it to Exchange.
```

Comment:
>  If you require your Product to be available for public consumption, you must publish it to Exchange.

Public consumption implies product will be available publically in reality it can be just kept in private Exchange. I think it should be reworded to 
For Product discoverability, you must publish it to Exchange.

Reply:
Done! Thanks!